### ISSUE TITLE
Officer-involved shootings

### ISSUE THREAD

**Issue by timwis** [MimiOnuoha/missing-datasets#5]
Hi - cool project! Looking forward to seeing it develop.

Regarding the first item on the list of missing datasets, I wanted to let you know that over the past year or two, [several cities](https://publicsafetydataportal.org/officer-involved-shooting-data/) have released data that describes officer-involved shootings. The data usually indicates whether the shooting resulted in a fatality. In Philadelphia, [our data release](https://www.opendataphilly.org/dataset/shooting-victims) includes _all_ shooting victims, whether or not an officer was involved, as well as a visualization (called a "Map Journal") that aims to contextualize the data alongside crime patterns (as much as it can at least).

**Comment by MimiOnuoha** [comment]
You're absolutely right! This is actually *not* a missing dataset, and hasn't been for awhile, though it's a wonderful example of something that once was. Will add a new category to reflect this point, thanks.
